webapp/channels/src/selectors/drafts.ts

```diff
@@ -124,7 +124,11 @@ export function makeGetDraft() {
         }
         const key = storageKey || `${prefixStorageKey}${suffixStorageKey}`;
 
-        const retrievedDraft = getGlobalItem<PostDraft>(state, key, DEFAULT_DRAFT);
+        let retrievedDraft = getGlobalItem<PostDraft>(state, key, DEFAULT_DRAFT);
+
+        if (retrievedDraft.metadata?.files) {
+            retrievedDraft = {...retrievedDraft, fileInfos: retrievedDraft.metadata.files};
+        }
```

Comment:
Does this end up returning a new draft on every call of the selector? Can we add tests to check that?

Reply:
Done, fixed.